src/Components/DD1 (2).json

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
Does this file have a use, @lambert3 ?

Reply:
yes it  does, its being used by the loading screen component